sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -169,24 +169,52 @@ namespace Azure { namespace Core { namespace Http { namespace Policies {
         NextHttpPolicy nextPolicy,
         Context const& context) const = 0;
 
-    /// Destructor.
+    /**
+     * @brief Destructs `%HttpPolicy`.
+     *
+     */
     virtual ~HttpPolicy() {}
 
     /**
-     * @brief Creates a clone of this HTTP policy.
-     * @return A clone of this HTTP policy.
+     * @brief Creates a clone of this `%HttpPolicy`.
+     * @return A clone of this `%HttpPolicy`.
      */
     virtual std::unique_ptr<HttpPolicy> Clone() const = 0;
 
   protected:
+    /**
+     * @brief Constructs a default instance of `%HttpPolicy`.
+     *
+     */
     HttpPolicy() = default;
+
+    /**
+     * @brief Constructs a copy of \p other `%HttpPolicy`.
+     * @param other Other `%HttpPolicy` to copy.
+     *
+     */
     HttpPolicy(const HttpPolicy& other) = default;
-    HttpPolicy(HttpPolicy&& other) = default;
+
+    /**
+     * @brief Assigns this `%HttpPolicy` to copy the \p other.
+     * @param other Other `%HttpPolicy` to copy.
+     * @return A reference to this `%HttpPolicy`.
+     *
+     */
     HttpPolicy& operator=(const HttpPolicy& other) = default;
+
+    /**
+     * @brief Contructs `%HttpPolicy` by moving \p other `%HttpPolicy`.
+     * @param other Other `%HttpPolicy` to move.
+     *
+     */
+    HttpPolicy(HttpPolicy&& other) = default;
   };
 
   /**
    * @brief The next HTTP policy in the stack sequence of policies.
+   * @note `%NextHttpPolicy` is an abstraction representing a next line in the stack sequence of
+   * policies, from the caller's perspective.
```

Comment:
This is a bit confusing to parse.

Maybe modify as something like this:
`...representing the next policy to execute in the sequence of policies.`

Reply:
I think to combine would be better "`is an abstraction representing the next policy in the stack sequence of policies, from the caller's perspective.`".

Reply:
Will bring in the follow up PR.

Reply:
What do you mean by "stack sequence of policies" and what does "the caller's perspective" mean? Those two phrases confuse me a bit, so I question their usefulness. Your call of course :)

Reply:
Maybe I am biased, because I think I know exactly what this means, and it makes perfect sense to me.
"next" policy is in relation to who? To the caller. I.e. whenever you see this and yo can call it, it is next relative to you, not in the absolute sense.
And as of the stack. Policies are a stack.

Reply:
I like the wording that I put in ##2378. If you give it to me, I'll leave it as I have it in that PR. But I don't feel super strongly on this, if you think of a better wording, please propose it in that PR.